src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -411,6 +412,9 @@ private void updatePatronTransactionOnRequestChange(RequestDTO requestDTO, InnRe
     if (requestDTO.getStatus() == CLOSED_CANCELLED &&
       (transaction.getState() == PATRON_HOLD || transaction.getState() == TRANSFER)) {
       log.info("Updating patron hold transaction {} on cancellation of a request {}", transaction.getId(), requestDTO.getId());
+      if (transaction.getState() != ITEM_SHIPPED) {
```

Comment:
If I understand correctly the first `if` statement already restricts transaction to have either PATRON_HOLD or TRANSFER states, so there is no chance that a transaction is in ITEM_SHIPPED state

Reply:
Thank you I had a  doubt